src/QueryStrategies/BatchQueryStrategies/TopKBatchQs.jl

```diff
@@ -0,0 +1,25 @@
+struct TopKBatchQs <: ExtendingBatchQs
```

Comment:
Isn't this just a special case of RandomBestBatchQs with m=0? If so, I'd call this "TopKBatchQs" and then have an optional constructor argument `(;m=0)`.